coderd/rbac/roles_test.go

```diff
@@ -248,6 +248,15 @@ func TestRolePermissions(t *testing.T) {
 				false: {otherOrgAdmin, otherOrgMember, memberMe, userAdmin},
 			},
 		},
+		{
+			Name:     "CreateCustomRole",
+			Actions:  []policy.Action{policy.ActionCreate},
+			Resource: rbac.ResourceAssignRole,
+			AuthorizeMap: map[bool][]authSubject{
+				true:  {owner},
+				false: {userAdmin, orgAdmin, orgMemberMe, otherOrgAdmin, otherOrgMember, memberMe, templateAdmin},
+			},
```

Comment:
👍  This is fine for now, but I can potentially see folks wanting to allow UserAdmins / OrgAdmins some level of control in future...

...maybe they just need to make a custom role? 😄

Reply:
Lol, yea I'm going to push this until I get to the UX. For this PR stack, I'm just getting custom roles to work.

Once I start building UX, I will revist who exactly should have this permission. But it could just be a custom role too as you said :man_shrugging: